src/routes/_static/config.js

```diff
@@ -0,0 +1 @@
+export const DEFAULT_INSTANCE_HOSTNAME = false
```

Comment:
`process.env.DEFAULT_INSTANCE_HOSTNAME` and inserting via Webpack would make it easier for instance admins to change this value